util/client.go

```diff
@@ -0,0 +1 @@
+package util
```

Comment:
?

Reply:
meant to delete, thanks for pointing out